lib/workers/repository/updates/flatten.ts

```diff
@@ -140,6 +140,7 @@ export async function flattenUpdates(
           packageFileConfig.lockFileMaintenance
         );
         lockFileConfig.updateType = 'lockFileMaintenance';
+        lockFileConfig.isLockFileMaintenance = true;
```

Comment:
@rarkins do we like it? It's probably easier for templates, but I like to reduce the computed properties. We really should fix our typings to catch those errors.

Reply:
I think is good for now